Misc/NEWS.d/next/Library/2019-06-18-16-29-31.bpo-37085.GeYaD6.rst

```diff
@@ -0,0 +1 @@
+Added support for the Linux SocketCAN Broadcast Manager API constants.
```

Comment:
Some BCM constants were already there; perhaps more precisely describe the nature of the added constants/flags.